gridt/src/app/home/expandable/expandable.component.scss

```diff
@@ -0,0 +1,58 @@
+.expand-wrapper {
+        height: 0px;
+        min-height: 20px;
+        max-height: 300px;
+        transition: 0.5s;
+    }
+.expanded {
+        min-height: 150px;
+}
+
+@media screen and (max-width: 280px) {
+        .expanded {
+                min-height: 310px;
+        }
+}
+
+@media screen and (min-width: 281px) and (max-width: 370px) {
+        .expanded {
+                min-height: 150px;
+
+        }
+}
+@media screen  and (min-width: 371px) and (max-width: 400px) {
+        .expanded {
+                min-height: 120px;
+        }
+} 
+@media screen  and (min-width: 401px) and (max-width: 500px) {
+        .expanded {
+                min-height: 110px;
+        }
+} 
+@media screen  and (min-width: 501px) and (max-width: 600px) {
+        .expanded {
+                min-height: 70px;
+        }
+} 
+@media screen  and (min-width: 601px) and (max-width: 800px) {
+        .expanded {
+                min-height: 100px;
+        }
+} 
+
+@media screen  and (min-width: 801px) and (max-width: 1500px) {
+        .expanded {
+                min-height: 120px;
+        }
```

Comment:
Is this even necessary? When I click on the message, I want to see the message. It should not take up more space than necessary.

Reply:
I agree that it shouldn't take up more room than necessary. That's why i added so many different versions. So that on each screen type the wasted space on expansion is minimized.

Reply:
Isn't the overflow visible class what makes the ... appear? Maybe you could just remove that class conditionally.

Reply:
```
.overflow-ellipsis {
        text-overflow: ellipsis;
        white-space: normal;
        overflow: hidden;
    }
```
    Should provide the '...' however they don't appear for me, whatever i do.
```
.overflow-visible {
        white-space: initial;
        
    }
```
Is what makes it that the text isn't cut off and continue on a new line.
If i remove the part commented on in the scss file than yes, it still expands. However the smooth transitional animation is thereby also removed as that only seems to work if the min-height changes.

So then the question becomes do we want easier to maintain or a nicer view experience for the user?